src/components/EventDate.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactElement } from "react";
+
+// Renders an event's day of the month numerically, month, and day of the week.
+export const EventDate = (props: {
+  startTime: Date;
+  live: boolean;
+}): ReactElement => {
+  const date = {
+    num: props.startTime.getDate(),
+    month: props.startTime.toLocaleString("en-US", {
+      month: "short",
+    }),
+    day: props.startTime.toLocaleString("en-US", {
+      weekday: "short",
+    }),
+  };
+
+  return (
+    <div>
+      {props.live ? (
+        <div className="event-date-red">LIVE</div>
```

Comment:
Not sure "event-date-red" needs to be its own @apply style in `styles.css` if we're not using it outside of the EventDate component, otherwise our `styles.css` file will get bloated pretty quickly. I think if we're only using it for the EventDate component we should keep the styling inline within this file and avoid having to clutter `styles.css`

Reply:
Same thing for `event-date-black` and `event-date-white`

Reply:
That's a great point lol - let me do that instead

Reply:
It might be more readable if define the styles as constants at the beginning of the component. For example

`const EventDateRed = "event-date bg-sga-red"`

and then use it as 

`<div className={EventDateRed}>`

Up to you though, I think either way works

Reply:
Or styled div 👀

Reply:
ooh that sounds good